kvmagent/kvmagent/plugins/ovn.py

```
@@ -382,6 +473,158 @@ def ovn_del_port(self, req):
+            r = controllerCtl.resetOvnControllerClusterIndex()
+            if r != 0:
+                rsp.success = False
+                rsp.error = "Failed to clear ovn-controller cache"
+                return jsonobject.dumps(rsp)
+
+        return jsonobject.dumps(rsp)
+
+    @kvmagent.replyerror
+    @bash.in_bash
+    def ovn_check_local_port(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.14.8)

Local variable `cmd` is assigned to but never used

Remove assignment to unused variable `cmd`

(F841)